x-pack/plugins/security_solution/public/app/deep_links/index.ts

```diff
@@ -289,38 +289,23 @@ export const securitySolutionsDeepLinks: SecuritySolutionDeepLink[] = [
           },
         ],
       },
-      {
-        id: SecurityPageName.case,
-        title: CASE,
-        path: CASES_PATH,
-        navLinkStatus: AppNavLinkStatus.visible,
-        features: [FEATURE.casesRead],
-        keywords: [
-          i18n.translate('xpack.securitySolution.search.cases', {
-            defaultMessage: 'Cases',
-          }),
-        ],
-        order: 9006,
-        deepLinks: [
-          {
-            id: SecurityPageName.caseCreate,
-            title: i18n.translate('xpack.securitySolution.search.cases.create', {
-              defaultMessage: 'Create New Case',
-            }),
-            path: `${CASES_PATH}/create`,
-            features: [FEATURE.casesCrud],
+      getCasesDeepLinks<SecuritySolutionDeepLink>({
+        basePath: CASES_PATH,
+        extend: {
```

Comment:
Should we pass also the `keywords`?

Reply:
in this case the deepLink `title` and the `keywords` are the same "Cases" word, so this keyword is not needed.